pynaco/naco.py

```diff
@@ -260,8 +258,8 @@ def normalize(input, leaveFirstComma, cleanUp=strict):
 
     # put it back to unicode so that we can match chars
     # input = unicode(input.decode('utf-8'))
-    if type(input) != UnicodeType:
-        input = unicode(input, 'utf-8')
+    if type(input) != str:

```

Comment:
Since we're already changing this, maybe we should update this to use the preferred type checking?
```suggestion
    if not isinstance(input, str):
```